src/assets/json/userSession.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
is this a mock?

Reply:
it looks like a mock for the json returned by `startSession` (in the app we use [a patched type](https://github.com/pagopa/io-app/blob/master/ts/types/pagopa.ts#L268))